examples/kubernetes/nfs-provisioner/nfs-server-lb.yaml

```diff
@@ -0,0 +1,42 @@
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: nfs-server
+  labels:
+    app: nfs-server
+spec:
+  type: LoadBalancer
+  selector:
+    app: nfs-server
+  ports:
+    - port: 2049
+      name: nfs-server
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+    name: nfs-server
+spec:
+    replicas: 1
+    selector:
+        matchLabels:
+            app: nfs-server
+    template:
+        metadata:
+            name: nfs-server
+            labels:
+                app: nfs-server
+        spec:
+            containers:
+                - name: nfs-server
+                  image: gcr.io/kubernetes-e2e-test-images/volume/nfs:1.0
+                  ports:
+                    - containerPort: 2049
+                  volumeMounts:
+                    - mountPath: /nfsshare
+                      name: data-volume
+            volumes:
+                name: data-volume
+                hostPath: /nfsshare-volume
```

Comment:
This means the nfs server is sticky to that Node right? If the pod gets rescheduled to another node, all the data is lost.

We should probably make it clear of this limitation and say this is for development purposes only.